### DISCUSSION TITLE
Has Termux updated 1.119.1?

### DISCUSSION THREAD

**Discussion by 743859910** [termux/termux-app#3845] [Q&A] [answered]
![Screenshot_2024-02-19-20-04-58-432_com.magicalstory.AppStore.jpg](https://github.com/termux/termux-app/assets/84552324/6cca700c-35b6-4ed6-b2d4-5af07d282449)

![Screenshot_2024-02-19-20-05-09-646_com.magicalstory.AppStore.jpg](https://github.com/termux/termux-app/assets/84552324/5110ed5b-4d53-4342-91d9-10b1058544fe)

**Comment by twaik** [comment, marked as answer]
No. It did not. The application store shown on your screenshot contains malicious apk.
